clang/lib/CConv/ConstraintBuilder.cpp

```diff
@@ -412,6 +413,46 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
   TypeVarInfo &TVInfo;
 };
 
+class PtrToStructDef : public RecursiveASTVisitor<PtrToStructDef> {
```

Comment:
Add comment to say what this does, and that `containsPtrToStructDef` is the entry point.